client/src/pages/workspace/board/addBoard/AddBoard.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import {IBoardEntity} from "../../../../model/entity/IBoardEntity";
+import {IBoardCreation} from "../../../../model/dto/IBoardCreation";
+import {AddBoardProp} from "./props";
+import Button from "@mui/material/Button";
+import "./style.css";
+import { TextField } from '@mui/material';
+
+const AddBoard: React.FC<AddBoardProp> = ({onCreate}: AddBoardProp) => {
```

Comment:
it would be better if you renamed the props and functions to explicit names. 
For example: `onCreate` is better to replace with `addBoard` and so on.